src/commands/aem/rde/reset.js

```diff
@@ -37,6 +46,16 @@ class ResetCommand extends BaseCommand {
 Object.assign(ResetCommand, {
   description: 'Reset the RDE',
   args: [],
+  flags: {
+    nowait: Flags.boolean({
```

Comment:
Could we please call this flag `wait` with a default of `true`, and add `allowNo: true` to the flag's configuration. This results in a flag called `--no-wait`, which makes for a consistent style. Note, the "log" command has a `--no-color` flag configured in the same way.

Reply:
agree. as this was merged already, I will open another PR and add you there.

Reply:
see https://github.com/adobe/aio-cli-plugin-aem-rde/pull/73